src/main/java/org/opentripplanner/graph_builder/module/time/TraficPredictionBuildeModule.java

```diff
@@ -0,0 +1,39 @@
+package org.opentripplanner.graph_builder.module.time;
+
+import org.opentripplanner.graph_builder.services.GraphBuilderModule;
+import org.opentripplanner.routing.edgetype.StreetEdge;
+import org.opentripplanner.routing.graph.Graph;
+
+import java.io.File;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.TreeMap;
+
+
+public class TraficPredictionBuildeModule implements GraphBuilderModule {
```

Comment:
`TraficPredictionBuildeModule` -> `TraficPredictionBuilderModule`

Reply:
fixed

Reply:
skoro już poprawiamy błędy, to `Traffic` a nie `Trafic` 👍

Reply:
done